apps/minaintyg/src/pages/Certificates/Certificates.tsx

```diff
@@ -18,6 +23,7 @@ export function Certificates() {
         intyg digitalt till Försäkringskassan och Transportstyrelsen. Läkarintyg om arbetsförmåga kan inte skickas digitalt till din
```

Comment:
Does this component have tests?

Reply:
Yes, tests are available here `apps\minaintyg\src\pages\Certificates\Certificates.test.tsx`